src/System.Runtime/tests/System/GuidTests.cs

```diff
@@ -336,6 +336,7 @@ public static void Equals(Guid guid1, object obj, bool expected)
                 Assert.Equal(expected, guid1.Equals(guid2));
                 Assert.Equal(expected, guid1 == guid2);
                 Assert.Equal(!expected, guid1 != guid2);
+                Assert.Equal(expected, guid1.GetHashCode().Equals(guid2.GetHashCode()));
```

Comment:
This is failing on netfx.  Presumably because of https://github.com/dotnet/coreclr/pull/5191 ?  Assuming it's expected this assert will need to be special-cased to execute on netfx.